forgettable-frontend/src/components/Dialog/Dialog.module.css

```diff
@@ -0,0 +1,55 @@
+.dialog {
+  position: fixed;
+  background: white;
+  width: 525px;
+  height: 400px;
+  z-index: 2;
+  border-radius: 4px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
There's a really useful mui (material ui) component called [Modal](https://mui.com/components/modal/#main-content). That will make css a lot more concise (no need to use fixed, top, left, translate, zindex as it automatically centers it for you!)